tools/loader/Loader.cpp

```diff
@@ -173,7 +173,7 @@ llvm::cl::list<std::string> doNotLowerNodesForProfilingOpt(
 llvm::cl::opt<std::string> ExecutionBackend(
     "backend",
     llvm::cl::desc("Backend to use, e.g., Interpreter, CPU, OpenCL:"),
-    llvm::cl::init("Interpreter"), llvm::cl::cat(loaderCat));
+    llvm::cl::init("CPU"), llvm::cl::cat(loaderCat));
```

Comment:
I'm a bit concerned about this specific change. Glow's CPU backend does not support all of the features provided by the Interpreter backend (e.g. fp16 support and some other features). Thus changing the default may have some unintended consequences.

Reply:
I can change it back if you want (not very important for me).

Reply:
Done. Changed the default backend from CPU back to Interpreter.